libraries/AP_Compass/CompassCalibrator.cpp

```diff
@@ -768,6 +768,23 @@ Vector3f CompassCalibrator::calculate_earth_field(CompassSample &sample, enum Ro
     return efield;
 }
 
+/*
+  return true if two rotations are equivalent
+  This copes with the fact that we have some duplicates, like ROLL_180_YAW_90 and PITCH_180_YAW_270
+ */
+bool CompassCalibrator::rotation_equal(enum Rotation r1, enum Rotation r2) const
```

Comment:
Why not make this part of AP_Math/rotations as a general helper? Seems useful in more locations then just here.